lib/quick-secrets/core.rb

```diff
@@ -7,14 +8,14 @@ class Core
     attr_reader :config
 
     def initialize
-
+      @gemspec = Gem::Specification::load("quick-secrets.gemspec")
+      @args = QuickSecrets::ParseArgs.new(@gemspec)
+      @config = QuickSecrets::Configuration.new(@args)
       @authenticator = QuickSecrets::Auth.new
       @secrets = QuickSecrets::Secret::Manager.new
-      @config = QuickSecrets::Configuration.new
 
-      puts "Initializing db..."
+      puts "initializing db #{@config['database']}"
```

Comment:
Some people use account details for the database connection string.
e.g. `postgres://user:admin@host:port`
 We don't want to print this

Reply:
agreed.